src/main/groovy/org/shipkit/internal/gradle/notes/ReleaseNotesPlugin.java

```diff
@@ -106,6 +104,6 @@ private static void configureDetailedNotes(final UpdateReleaseNotesTask task,
         task.setPreviousVersion(project.getExtensions().getByType(VersionInfo.class).getPreviousVersion());
 
         task.setReleaseNotesData(releaseNotesFetcher.getOutputFile());
-        task.setContributorsDataFile(contributorsFetcher.getOutputFile());
+        task.setContributorsDataFile(contributorsFetcher.getOutputs().getFiles().getSingleFile());
```

Comment:
I changed `contributorsFetcher` to type `org.gradle.api.Task` since we might integrate more systems in the future (and furthermore the output is available via the gradle api).
You are right, using `outputFile` is way simpler.